web/source/keymanweb.js

```diff
@@ -3054,9 +3114,14 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
             keymanweb.moveToNext(false);
           return true;        
                   
-        case 16: 
+        case 16: //"K_SHIFT":16,"K_CONTROL":17,"K_ALT":18
         case 17: 
-        case 18: keymanweb._NotifyKeyboard(Levent.Lcode,Levent.Ltarg,0); return osk._UpdateVKShift(Levent, Levent.Lcode-15, 1);  // I2187
+        case 18: 
+        case 20: //"K_CAPS":20, "K_NUMLOCK":144,"K_SCROLL":145
```

Comment:
Are these vk codes consistent on Mac and Linux as well? (and all browsers)?

Reply:
The vk codes are confirmed to be Hackintosh-approved, at the least.

Looking at the [MDN keycode table](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), we can rely on these modifier codes.  The one problem would be for Linux when AltGraph is a thing.

Reply:
Hmm, that table is only listing desktop OSes, with nothing for mobile.  Android and iOS may not match up; that'll take further testing.